db/schema.rb.example

```diff
@@ -49,6 +49,7 @@ ActiveRecord::Schema.define(:version => 20170824172336) do
     t.string  "mail",      :limit => 64
     t.string  "homepage"
     t.integer "aid",                             :default => 0,  :null => false
+    t.string  "reference"
```

Comment:
Here I believe you have to update the timestamp at the top of the file to match the latest migration version -- that may be what's stopping you!

Reply:
Oh thanx. Corrected this